app/src/main/java/com/redcoracle/episodes/RefreshShowUtil.java

```diff
@@ -41,12 +42,11 @@ public static void refreshShow(int showId, ContentResolver contentResolver) {
 
 		Log.i(TAG, String.format("Refreshing show %d", showId));
 
+		SharedPreferences preferences = Preferences.getSharedPreferences();
+		preferences.getString("pref_language", "en");
```

Comment:
I think this line is not needed?

Reply:
Oh, yes. I must have used it for debugging. Thanks!